.github/workflows/phpunit.yaml

```diff
@@ -0,0 +1,97 @@
+name: PHPUnit
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  phpunit:
+    name: "phpunit - php ${{ matrix.php-version }}"
+    runs-on: ubuntu-20.04
+    continue-on-error: false
+    env:
+      SYMFONY_REQUIRE: ${{matrix.symfony-require}}
+      SYMFONY_DEPRECATIONS_HELPER: ${{matrix.symfony-deprecations-helper}}
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.4"
+          - "8.0"
+          - "8.1"
+        deps:
+          - "normal"
+        symfony-require:
+          - ""
+        symfony-deprecations-helper:
+          - ""
+        include:
+
+          # Test against stable Symfony 5.3
+          - symfony-require: "5.3.*"
```

Comment:
Looks like the lowest Symfony version we test here is `5.3.*`... I wonder if we should test this bundle on Symfony 4 too (or instead that `5.3.*`), e.g. `4.4.*` at least since we say we support that version in the composer.json file. Thoughts?

Reply:
Symfony will end support for 5.3 in 11 days.  I would imagine that anyone doing active development is at least trying to to bump to 5.4, in preparation for 6.0.  But obviously if everything still works in 5.3 we should support it, but if supporting an unmaintained version is blocking supporting the current version (6.0), I think the current version should take precedence.

Reply:
IMO this bundle should still support 4.4 LTS for which the end of bug fixes will be on November 2022, and the end of security fixes - on November 2023, for more info see: https://symfony.com/releases/4.4

So, I think this bundle should test Symfony 4.4 as the lowers Symfony version supported here

Reply:
That makes sense.  4.4 has a way to handle all the new signatures from 6.0, as well as the deprecated ones, so there shouldn't be any issues with supporting 4.4, 5.4, and 6.0.  

Can you make the changes for that, then release this?